noetic/distribution.yaml

```diff
@@ -7589,11 +7589,19 @@ repositories:
       version: release/0.6.x
     status: maintained
   pybind11_catkin:
+    doc:
+      type: git
+      url: https://github.com/wxmerkt/pybind11_catkin.git
```

Comment:
This is all fine - but by policy we like to have a `LICENSE` file in the repo. You don't need to re-release, but can you add the BSD license into your repo?

Reply:
Thanks for flagging this up - just added the BSD-3 license as per package.xml to the source repo